scripts/ci/upload-release

```diff
@@ -47,6 +47,19 @@ DCOS_BRANCH="dcos-ui/${PKG_TARGET}+dcos-ui-latest"
 ## Create Assets & Generate Upload Data
 #####################################################################
 
+VERSION_PLACEHOLDER="0.0.0-dev+semantic-release"
```

Comment:
@DanielMSchmidt added

Reply:
we could read it from `package.json` as we do with `PKG_TARGET` and `.dcosrc` above :trollface: